src/nunavut/version.py

```diff
@@ -3,6 +3,6 @@
 # This software is distributed under the terms of the MIT License.
 #
 
-__version__ = "1.2.2"
+__version__ = "1.2.3"
```

Comment:
```suggestion
__version__ = "1.3.0"
```

You probably meant this. The minor version comes after the major version, then comes the patch version.